plugins/PromisedScene/main.js

```diff
@@ -369,7 +400,187 @@ module.exports = async ({
       result.studio = askedStudio;
     }
 
-    return result;
+    $log("====  Final Entry =====");
+
+    for (const property in result) {
+      if (
+        property === "actors" &&
+        result[property] === "" &&
+        Array.isArray(actor) &&
+        actor.length
+      ) {
+        result[property] = actor;
+      }
+      if (
+        property === "studio" &&
+        result[property] === "" &&
+        Array.isArray(studio) &&
+        studio.length
+      ) {
+        result[property] = studio;
+      }
+      if (property === "releaseDate") {
+        $log(`${property}: ${util.timeConverter(result[property])}`);
+      } else {
+        $log(`${property}: ${result[property]}`);
+      }
+    }
+
+    const { CorrectImportInfo: resultsConfirmation } = await questionAsync({
+      type: "input",
+      name: "CorrectImportInfo",
+      message: "Is this the correct scene details to import? (y/N)",
+      testAnswer: testMode ? testMode.CorrectImportInfo : "",
+    });
+
+    const ResultsImport = util.isPositiveAnswer(resultsConfirmation);
+    if (ResultsImport) {
+      return result;
+    } else {
+      const res = await makeChoices();
+      return res;
+    }
+  }
+
+  /**
+   * Provides 3 choices to be completed:
+   * 1) search again => Completes a manual Search of The Porn Database
+   * 2) manual info => plugin returns "manualImport"
+   * 3) Do nothing => plugin does not return any results (empty object)
+   * @returns {Promise<object>} ???
+   */
```

Comment:
Super nitpicky, if you use markdown list style, vscode should interpet it properly
![image](https://user-images.githubusercontent.com/17180727/89660264-67d50680-d8d1-11ea-82bf-27df262c0fc0.png)
vs
![image](https://user-images.githubusercontent.com/17180727/89660351-863b0200-d8d1-11ea-85ef-09a8bfcdd62c.png)
```suggestion
  /**
   * Provides 3 choices to be completed:
   * 1. search again => Completes a manual Search of The Porn Database
   * 2. manual info => plugin returns "manualImport"
   * 3. Do nothing => plugin does not return any results (empty object)
   * @returns {Promise<object>} ???
   */
```